lab-elizabeth/app/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html ng-app="demoApp">
+  <head>
+    <title> Cowsay Client </title>
+    <link href="https://fonts.googleapis.com/css?family=Nova+Cut|Nova+Flat|Nova+Slim|Nova+Square" rel="stylesheet">  </head>
+  <body>
+    <nav ng-controller="NavController as navCtrl">
+      <i class="fa fa-hand-spock-o fa-4x"></i>
+      <ul>
+        <li ng-repeat="navItem in navCtrl.routes">
+          <a ng-href="{{navItem.url}}">{{navItem.name}}</a>
+        </li>
+      </ul>
+      <i class="fa fa-hand-lizard-o fa-4x"></i>
+    </nav>
+
+    <section ng-controller="CowsayController as cowsayCtrl" ng-init="cowsayCtrl.currentCow = 'eyes'">
+      <h1> {{cowsayCtrl.title}} </h1>
+      <form ng-submit="cowsayCtrl.speak(cowsayCtrl.text)" novalidate name="cowsayForm" class="form-horizontal">
```

Comment:
The assignment (lab 22) asked you to add an ng-class to your form if it's invalid.